apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -3,15 +3,40 @@ import { Button, Checkbox, DataTable, Stack } from "@cc/ui-chakra";
 import useGetOrders, { Order } from "@src/services/orders/useOrders";
 import dayjs from "dayjs";
 import { createColumnHelper } from "@tanstack/react-table";
+import { HStack, VStack } from "@cc/ui-chakra";
 
 const OpenOrdersList = () => {
-  const { lists } = useGetOrders();
+  const { isLoading, handleAcceptOrderSubmit, lists } = useGetOrders();
   const columnHelper = createColumnHelper<Order>();
+  const [checkedItems, setCheckedItems] = React.useState([false]);
+  const allChecked = checkedItems.every(Boolean);
+  const isIndeterminate = checkedItems.some(Boolean) && !allChecked;
+
+  React.useEffect(() => {
+    (checkedItems.length = lists.length), checkedItems.fill(false, 0, lists.length);
+  }, [lists]);
 
   const columns = [
     columnHelper.accessor("id", {
-      cell: (info) => <Checkbox key={info.getValue()} value={info.getValue()}></Checkbox>,
-      header: () => <Checkbox></Checkbox>,
+      cell: (info) => (
+        <Checkbox
+          onChange={(e) => {
+            const list = checkedItems;
+            checkedItems[info.row.index] = e.target.checked;
+            setCheckedItems([...list]);
```

Comment:
If we are mutating states involving previous states, it's better we do 

```ts
setCheckedItems((prev) => [...prev, (prev[info.row.index] = e.target.checked)]);
```

https://reactjs.org/docs/hooks-reference.html#functional-updates

And don't mutate `checkedItems` then update states (or any variables really) with the reference (`list`) to that mutated variable~~